plugin/pkg/admission/network/defaultingressclass/admission_test.go

```diff
@@ -142,8 +166,8 @@ func TestAdmission(t *testing.T) {
 			classes:         []*networkingv1.IngressClass{defaultClass1, defaultClass2, classWithFalseDefault, classWithNoDefault, classWithEmptyDefault},
 			classField:      nil,
 			classAnnotation: nil,
-			expectedClass:   nil,
-			expectedError:   errors.NewForbidden(networkingv1.Resource("ingresses"), "testing", errors.NewInternalError(fmt.Errorf("2 default IngressClasses were found, only 1 allowed"))),
+			expectedClass:   utilpointer.StringPtr(defaultClass1.Name),
+			expectedError:   nil,
```

Comment:
minor: we don't need to set this field to nil - it's just noise

Reply:
May I ask which field you are talking about? If it is `expectedError`, we have to set it to nil,since we no longer return error when there are two or more ingressClasses.

Reply:
not setting the field, leaving it empty, has the same effect to set it to nil
```suggestion
```

Reply:
but it was already this way, and seems there are other test cases setting it

Reply:
> 

Yes,And I set the field to nil in order to keep the format uniform, should we delete the nil field for all the tests?

Reply:
I would normally argue to remove it all over, but that can be a different PR